calcom/cal.com
# fix: ignore original rescheduled booking for booking limits

## What does this PR do?

Fixes that when a booking is rescheduled it will not be counted as an existing booking for the booking limits. 

So if there is a limit of 1 booking a day, the user can still reschedule to the same day. 

Fixes #12347

## Type of change


## How should this be tested?

- Create event type with booking limit 1 per day 
- Make a booking 
- Reschedule booking and see if you have available slots on the days of the original booking
